newrelic.yml

```diff
@@ -406,6 +406,24 @@ common: &default_settings
   # Controls auto-instrumentation of the Thread class at start up to automatically add tracing to all Threads created in the application.
   # instrumentation.thread.tracing: false
 
+  # Controls auto-instrumentation of gRPC clients at start up.
+  # May be one of [auto|prepend|chain|disabled].
+  # instrumentation.grpc_client: auto
+
+  # Controls auto-instrumentation of gRPC servers at start up.
+  # May be one of [auto|prepend|chain|disabled].
+  # instrumentation.grpc_server: auto
+
+  # Specifies a list of hostname patterns separated by commas that will match
+  # gRPC hostnames that traffic is to be ignored by New Relic for.
+  # New Relic's gRPC client instrumentation will ignore traffic streamed to a
+  # host matching any of these patterns, and New Relic's gRPC server
+  # instrumentation will ignore traffic for a server running on a host whose
+  # hostname matches any of these patterns. By default, no traffic is ignored
+  # when gRPC instrumentation is itself enabled.
+  # For example, "private.com$,exception.*"
```

Comment:
Could you make `newrelic.yml` and this match their descriptions?

Reply:
`default_source.rb` addressed with 3f5f9af98a932478ebcc264689da9bb278986607